publish/pub.go

```diff
@@ -70,11 +70,12 @@ func NewPublisher(info beat.Info, pipeline beat.Pipeline, shutdownTimeout time.D
 		WaitClose: shutdownTimeout,
 		Fields: common.MapStr{
 			"observer": common.MapStr{
-				"type":         info.Beat,
-				"hostname":     info.Hostname,
-				"version":      info.Version,
-				"id":           info.ID.String(),
-				"ephemeral_id": info.EphemeralID.String(),
+				"type":          info.Beat,
+				"hostname":      info.Hostname,
+				"version":       info.Version,
+				"version_major": 7,
```

Comment:
why this is hardcoded and not derived from info.Version?

Reply:
I assumed it was for performance reasons.

Reply:
performance of `strings.Split("info.Version", ".")[0]`?
ok, was just curious. i guess this code won't be here by 8.0 :+1:

Reply:
This was to avoid any chance of a runtime error.  A test was added so we don't forget to change this for 8.0.